src/main/java/com/iexec/common/utils/purge/ExpiringTaskMapFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.iexec.common.utils.purge;
```

Comment:
I think another package name would be better
What do you think of `com.iexec.common.lifecycle.purge` ?
I don't really feel it is a good idea to add another package under the already multi-purpose `utils` package.